README.md

```diff
@@ -17,19 +45,31 @@ In addition to `miriway-shell`, Miriway has:
 * A default configuration file `miriway-shell.config`; and,
 * A greeter configuration `miriway.desktop` so Miriway can be selected at login
 
-Miriway has been tested with shell components from several desktop environments 
-and there are notes on enabling these in `miriway-shell.config`.
+## Pre-packaged Miriway
+
+Miriway is available as a Snap, which provides a basic default configuration 
+and also includes the example configuration scripts to help set up a variety
+of options.
 
-## In development
+[![miriway](https://snapcraft.io/miriway/badge.svg)](https://snapcraft.io/miriway)
```

Comment:
It is also available in a variety of distributions, notably Fedora. 😄 

You can also add this to show that in the README:
```
[![Packaging status](https://repology.org/badge/vertical-allrepos/miriway.svg)](https://repology.org/project/miriway/versions)
```

Which will look like so:

[![Packaging status](https://repology.org/badge/vertical-allrepos/miriway.svg)](https://repology.org/project/miriway/versions)

Reply:
Cool, thanks!

I had guessed there was a way to do that, but hadn't found it